src/LiteCore/parse/templates_c4/C4SocketFactory.cs

```diff
@@ -1,4 +1,4 @@
-    internal unsafe struct C4SocketFactory
+    internal unsafe partial struct C4SocketFactory
```

Comment:
What is the purpose of making this partial (and the others that were made partial)?

Reply:
I thought they are generated by binding script?

Reply:
This one is a template, and the binding script configuration was also modified as part of this PR.  I'm just wondering if there is a reason to change them.

Reply:
There is no reason to change the script anymore. Let me try to revert them all.